src/libraries/System.Composition.AttributedModel/src/System.Composition.AttributedModel.csproj

```diff
@@ -1,7 +1,8 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFrameworks>netstandard2.0;net461</TargetFrameworks>
+    <TargetFrameworks>$(NetCoreAppCurrent);netstandard2.0;net461</TargetFrameworks>
```

Comment:
Question unrelated to your PR: doesn't nuget understand .NET 4.6.1 to support .NET Standard 2.0?  Wondering why in several libs we need both assets.

Reply:
Yes it does, however if we don't have a net461 asset, the customers when using any of these packages for net461 would get all the netstandard shims for type unification and would need binding redirects which is painful for customers. We were getting a lot of issues and pain and that's why we decided to add the net461 assets to our packages to help ease that pain.

Reply:
.NET Framework 4.6.1 reaches end of support in April 2022:
https://devblogs.microsoft.com/dotnet/net-framework-4-5-2-4-6-4-6-1-will-reach-end-of-support-on-april-26-2022/
I guess we'll consider removing these assets for .NET 7 then?

Reply:
Correct, for NET7 we will upgrade all net461 assets to net462.